src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -267,9 +267,13 @@ case class Seqn(stmts: Vector[Stmt])(val info: Source.Parser.Info) extends Stmt
 
 case class Label(id: LabelProxy)(val info: Source.Parser.Info) extends Stmt
 
+case class Continue(label: Option[String], escLabel: String)(val info: Source.Parser.Info) extends Stmt
```

Comment:
what does `escLabel` mean? I would either expand the parameter name to make it self explanatory or add a JavaDoc comment